integrations/phishing_analyzers/analyzers/seleniumwire_request_serializer.py

```diff
@@ -0,0 +1,92 @@
+import base64
+from datetime import datetime
+
+from seleniumwire.request import Request, Response, WebSocketMessage
+
+
+def dump_seleniumwire_requests(request: Request) -> dict:
```

Comment:
Added these logs. I would avoid logging all request for sizing reason. What do you think?

Reply:
yep I meant adding the information regarding when this method has been accessed (this is mandatory for tracking purposes). The dump of the dictionary should not be logged because noisy